.travis.yml

```diff
@@ -18,13 +18,17 @@ install:
   # Install dependencies
   - conda create -n test-environment python=$TRAVIS_PYTHON_VERSION
   - source activate test-environment
-  - conda install pytest numpy pip coverage toolz pandas cython scikit-learn cytoolz dill networkx pyzmq psutil
+  - conda install pytest numpy pip coverage toolz pandas cython scikit-learn cytoolz dill networkx pyzmq psutil ipython
   - pip install chest
   - pip install git+https://github.com/blosc/bcolz --upgrade
 
   # Install dask
   - python setup.py install
 
+  # start the ipcluster, it takes a while to start
+  - ipcluster start -n 4 &
+  - sleep 20
```

Comment:
https://github.com/ipython/ipyparallel/blob/master/ipyparallel/tests/clienttest.py

Had to go find this in the fancy new split org.  I'm star number 2!